tests/puppeteer/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "eslint  --fix --ignore-path .gitignore .",
     "specific-test": "./node_modules/mocha/bin/mocha campaigns/$TEST_PATH",
     "sanity-tests": "./node_modules/mocha/bin/mocha campaigns/sanity/*",
-    "sanity-travis": "./node_modules/mocha/bin/mocha --bail campaigns/sanity/*",
+    "sanity-travis": "node --version && ./node_modules/mocha/bin/mocha --bail campaigns/sanity/*",
```

Comment:
Why checking node version here?

Reply:
I debug, I have a problem only on travis 
![image](https://user-images.githubusercontent.com/48441421/80987220-00bb3800-8e32-11ea-8c7e-c170f54ddf55.png)
and I think its linked to node version
puppeteer v3 support node 10 and we have only node 8 on travis.
I don't know if we can change it ?

Reply:
I tried to use nvm but not found, @PierreRambaud can you tell me we can switch to node 10 on travis ?

Reply:
A possible way: https://medium.com/@_Ma27_/travis-ci-with-php-and-nodejs-done-right-57bd4d650f75
We are using the PHP container, you need to install nodejs manually :/

Reply:
Thanks @PierreRambaud , this works